src/presentation/http/router/note.ts

```diff
@@ -26,10 +53,38 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
   const noteService = opts.noteService;
 
   /**
-   * Get notes
+   * Get note by id
    */
-  fastify.get('/', async () => {
-    return { hello: 'world' };
+  fastify.get<{ Params: GetNoteByIdOptions }>('/:id', async (request, reply) => {
+    const params = request.params;
+
+    /**
+     * TODO: Validate request params
+     */
+    const { id } = params;
+
+    const note = await noteService.getNoteById(id);
+
+    /**
+     * Check if note does not exist
+     */
+    if (!note) {
+      const response: ErrorResponse = {
+        code: StatusCodes.NOT_FOUND,
```

Comment:
call this field `status`. `code` is our custom error code used for translation

Reply:
done